packages/web-components/fast-foundation/src/toolbar/toolbar.ts

```diff
@@ -198,6 +189,31 @@ export class Toolbar extends FoundationElement {
         return true;
     }
 
+    /**
+     * get all the slotted elements
+     * @internal
+     */
+    protected getAllSlottedItems(): (HTMLElement | Node)[] {
+        return [
+            ...this.start.assignedNodes(),
+            ...this.slottedItems,
+            ...this.end.assignedNodes(),
+        ];
+    }
```

Comment:
* This can be a getter
* switching to `assignedElements()` means we don't have to handle weird types or worry about text nodes

```suggestion
    protected get allSlottedItems(): Element[] {
        return [
            ...this.start.assignedElements(),
            ...this.slottedItems,
            ...this.end.assignedElements(),
        ];
    }
```

Reply:
awesome! updated to this :)